src/MiniCover/Program.cs

```diff
@@ -132,10 +132,17 @@ static int Main(string[] args)
                     UpdateWorkingDirectory(workDirOption);
 
                     var coverageFile = GetCoverageFile(coverageFileOption);
-                    var result = LoadCoverageFile(coverageFile);
-
-                    if (File.Exists(result.HitsFile))
-                        File.Delete(result.HitsFile);
+                    try
```

Comment:
I wonder if it's better to use a File.Exists instead of try catch. What do you think?

Reply:
Yes, I really should try to read code while still half asleep :)